hazelcast/test/benchmark/README.md

```diff
@@ -0,0 +1,24 @@
+#Dependencies:
+    - [google benchmark](https://github.com/google/benchmark): Mac OS: brew install google-benchmark
+    - hazelcast-cpp-client 
+    - Boost
+
+#Build Instructions:
+mkdir build
+cd build
+cmake .. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_PREFIX_PATH=/path/to/boost/install/dir;/path/to/hazelcast-cpp-client/install/dir
```

Comment:
I have tried to follow the instructions 
And I run the following

cmake .. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_PREFIX_PATH=/usr/local/Cellar/boost/1.75.0_2;/Users/sancar/projects/hazelcast-cpp-client

And it could find Thrift.
```
  By not providing "FindThrift.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Thrift", but
  CMake did not find one.

  Could not find a package configuration file provided by "Thrift" (requested
  version 0.13.0) with any of the following names:

    ThriftConfig.cmake
    thrift-config.cmake

  Add the installation prefix of "Thrift" to CMAKE_PREFIX_PATH or set
  "Thrift_DIR" to a directory containing one of the above files.  If "Thrift"
  provides a separate development package or SDK, be sure it has been
  installed.
```

Note that I can build the client itself with tests, so thrift should be there.

Reply:
Does this needs to build the tests of hazelcast-cpp-client ? I wonder why it is trying to find Thrift

Reply:
It shouldn't need Thrift. 
Is the path `/Users/sancar/projects/hazelcast-cpp-client` contain the repository, or is it an install directory?

Reply:
did you change to the benchmark directory `cd hazelcast/test/benchmark/`?

Reply:
Ok, I have managed to make it work by correctly installing `hazelcast-cpp-client`